website/docs/r/nsxt_app_port_profile.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_app_port_profile"
+sidebar_current: "docs-vcd-resource-nsxt-app-port-profile"
+description: |-
+  Provides a resource to manage NSX-T Application Port Profiles. Application Port Profiles include a
+  combination of a protocol and a port, or a group of ports, that is used for firewall and NAT
+  services on the edge gateway. In addition to the default port profiles that are preconfigured for
```

Comment:
Just looking at the paragraph's capitalization, would be nice to capitalize all major keywords (same ask for same paragraph in the body below too):
```suggestion
  services on the Edge Gateway. In addition to the default port profiles that are preconfigured for
```

Reply:
Fixed